### DISCUSSION TITLE
how can I add one word to dictionary

### DISCUSSION THREAD

**Discussion by mohammadfrh** [AnySoftKeyboard/AnySoftKeyboard#3621] [General]
hi everyone.
I want to add just one custom word for show suggestion if the user type in keyboard.
I can not open .dict file ,there are symbol and encrypt char , but I can see and edit Prebuild file and after edit file suggestion not add.
how can I open and edit .dict file?

**Comment by FabulousOne** [comment]
You can just open the ASK app, go to speech - dictionaries. There you can add more words.
Or just tap twice on the word you wrote in your prediction bar.
